src/organizations/organizations.spec.ts

```diff
@@ -37,91 +42,75 @@ describe('Organizations', () => {
 
     describe('with the domain option', () => {
       it('forms the proper request to the API', async () => {
-        mock
-          .onGet('/organizations', {
-            domains: ['example.com'],
-          })
-          .replyOnce(200, listOrganizationsFixture);
+        fetchOnce(listOrganizationsFixture);
 
         const { data } = await workos.organizations.listOrganizations({
-          domains: ['example.com'],
+          domains: ['example.com', 'example2.com'],
         });
 
-        expect(mock.history.get[0].params).toEqual({
-          domains: ['example.com'],
+        expect(fetchSearchParams()).toEqual({
+          domains: 'example.com,example2.com',
```

Comment:
This is one meaningful change here that I'd like to call out.

I have changed the output here because `axios` would serialize arrays automatically, and here we use `URLSearchParams` which can't do that, but do serialize using comma.

So with a single domain, I would end up with just `example.com` as a string. So I added a second one and updated the equal check so it's easier to understand what the SDK produces.

I have checked our API controllers and wherever we expect arrays, we do have a transform that works with both arrays and strings:

```tsx
@Transform(({ value }: { value: string | string[] }) => {
    if (Array.isArray(value)) {
      return value;
    }

    return value.split(',');
  })
```

I've also validated this all works using a local build of the new package against the Open Source AuthKit repo.